client/src/services/requests-service.js

```diff
@@ -20,4 +20,18 @@ export const handlePost = (endpoint, body) => {
         body: JSON.stringify(body)
     }
     return fetch(url, requestParams)
+}
+
+export const handlePut = async (endpoint, body) => {
```

Comment:
You should use Axios for FE requests. More reliable with better error handling than fetch. Its kind of a standard.